### DISCUSSION TITLE
Issue With Buy Button Not Working

### DISCUSSION THREAD

**Discussion by BarryHewitt** [vercel/next.js#28179] [Help]
Good afternoon all, for some reason the buy button does not work on this app and I just cannot seem to figure out why, when you click it my wallet loads as expected but when i hit the buy button nothing happens, my full repo can be found here, i'm so close to having this finished, any help would be greatly appreciated

https://github.com/BarryHewitt/Artesia

**Comment by eigenpaul** [comment]
The repo you shared is either private or does not exist. I would be more than happy to help, given you share the actual code
